test/py/Makefile.am

```diff
@@ -71,50 +71,50 @@ TESTS= $(py_unit_tests)
 
 
 gen-py/%/__init__.py: ../%.thrift $(THRIFT)
-	test -f ../v0.16/$(notdir $<) \
-	&& $(THRIFT) --gen py  ../v0.16/$(notdir $<) \
+	test -f ../$(notdir $<) \
```

Comment:
we shouldn't change this one. the ones under `v0.16` are for the languages not have uuid implemented yet (and python is one of them).

Reply:
@jimexist I see that you force-pushed on this PR. Did you address the section @fishy mentioned?

Reply:
no not really - i didn't spend time in python impl in the last year or so